pkg/mattermost/mattermost.go

```diff
@@ -176,10 +176,7 @@ func (b *Bot) getChannel() *model.Channel {
 	// Checking if channel exists
 	botChannel, resp := client.GetChannelByName(b.ChannelName, b.getTeam().Id, "")
 	if resp.Error != nil {
-		// Creating channel
-		if botChannel, resp = client.CreateChannel(mmChannel(b.ChannelName, b.getTeam().Id)); resp.Error != nil {
-			logging.Logger.Fatal("There was a problem finding channel in Mattermost. Error: ", resp.Error)
-		}
+		logging.Logger.Fatal("There was a problem finding channel in Mattermost. Error: ", resp.Error)
```

Comment:
Dump all contextual data in error. Channel  name etc.

Reply:
Made required changes.